doc/source/index.rst

```diff
@@ -5,7 +5,7 @@
 .. note:: Adapting to a new version?
 
    If you are adapting to a new version of the actions, please refer to the
-   :ref:`migration guide <migration_guide>`.
+   :ref:`migration_guide`.
 
 For more information on available actions and how to use them,
 see each card below:
```

Comment:
```suggestion
see each of these cards:
```
Don't use positional adverbs because in the PDF the list may not appear below but on the next page.